huxley/www/static/js/huxley/components/AdvisorRosterView.js

```diff
@@ -109,6 +114,51 @@ var AdvisorRosterView = React.createClass({
     }.bind(this));
   },
 
+  _addDelegatePressed: function() {
+    return (
+      <form>
+        Name: <input type="text" placeholder="Name" valueLink={this.linkState('name')} /><br>
```

Comment:
`br` is a self closing element. Should be `<br />` . That should fix your compiler error you were mentioning